PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 ## Summary (required)
 
-- Resolves #[_Issue number_]
+- Resolves #Issue number
```

Comment:
I suggest making this Issue_number (with an underscore), so it's easier to replace the entire phrase by just double clicking on the string

Reply:
@patphongs sure! Made the change here: 59a81068c856d989a07dd1e09af357d0bd764b44